packages/components/src/search/style.scss

```
@@ -10,6 +10,14 @@
 		fill: $core-grey-light-900;
 	}
 
+	.woocommerce-tag {
+		max-width: 100%;
+	}
+
+	.woocommerce-tag .woocommerce-tag__text {
+		max-width: calc( 100% - 24px );
```

Our linter (`npm run lint:css`) will complain about the spaces inside the parenthesis.
```suggestion
		max-width: calc(100% - 24px);
```